dep11/generator.py

```diff
@@ -274,6 +296,8 @@ def handle_error(e):
 
             log.info("Completed metadata extraction for suite %s/%s" % (suite_name, component))
 
+        langpacks.cleanup()
```

Comment:
I agree with you on that, also having it as a context manager in this case wouldn't have any benefit over explicitly cleaning up at the end, or would it?